packages/gatsby-transformer-remark/src/extend-node-type.js

```diff
@@ -193,7 +205,7 @@ module.exports = (
           return Promise.resolve()
         }
       })
-      const markdownAST = remark.parse(markdownNode.internal.content)
+      const markdownAST = remark.parse(content)
```

Comment:
Related to the previous comment. This will ignore changes applied by `mutateSource` above (as they are applied to `markdownNode.internal.content` not to `content`)